e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,173 @@
+
+        // * The last message is written by user1 and contains the correct text.
+        cy.getLastPost().should('contain', 'Second test').and('contain', user1.username);
+    });
+
+    it('MM-T688 /expand', () => {
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl1);
+
+        // # Post message
+        cy.postMessage('http://www.mattermost.org/wp-content/uploads/2016/04/icon_WS.png');
+        cy.getLastPostId().as('postID');
+
+        // # Post command
+        cy.postMessage('/expand');
+
+        cy.get('@postID').then((postID) => {
+            cy.get(`#post_${postID}`).should('be.visible').within(() => {
+                // * Preview should be expanded
+                cy.findByLabelText('Toggle Embed Visibility').
+                    should('be.visible').and('have.attr', 'data-expanded', 'true');
+
+                // * Preview should be visible
+                cy.findByLabelText('file thumbnail').should('be.visible');
+            });
+        });
+
+        // * System post received confirming the new setting
+        cy.getLastPost().should('contain', 'Image links now expand by default').and('contain', 'System');
+    });
+
+    it('MM-T689 capital letter autocomplete, /collapse', () => {
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl1);
+
+        // # Post message
+        cy.postMessage('http://www.mattermost.org/wp-content/uploads/2016/04/icon_WS.png');
+        cy.getLastPostId().as('postID');
+
+        // # Open RHS
+        cy.clickPostCommentIcon();
+
+        // # Type uppercase letter
+        cy.get('#reply_textbox').type('/C');
+
+        // # Scan inside of suggestion list
+        cy.get('#suggestionList').should('exist').and('be.visible').within(() => {
+            // * Verify the collapse option exist in autocomplete and select it
+            cy.findAllByText('collapse').first().should('exist').click();
+        });
+
+        // # Hit enter to send the message
+        cy.get('#reply_textbox').type('{enter}');
+
+        cy.get('@postID').then((postID) => {
+            cy.get(`#rhsPost_${postID}`).should('be.visible').within(() => {
+                // * Preview should not be expanded
+                cy.findByLabelText('Toggle Embed Visibility').
+                    should('be.visible').and('have.attr', 'data-expanded', 'false');
+
+                // * Preview should not be visible
+                cy.findByLabelText('file thumbnail').should('not.be.visible');
+            });
+        });
+
+        // * System post received confirming the new setting
+        cy.getLastPost().should('contain', 'Image links now collapse by default').and('contain', 'System');
+    });
+
+    it('MM-T705 Ephemeral message', () => {
+        cy.apiLogin(user1);
+        cy.visit(testChannelUrl1);
+
+        // # Navigate to slash commands and create the slash command
+        cy.get('#headerInfo').click();
+        cy.get('#integrations').click();
```

Comment:
This is failing on me since "Integrations" option is not available to `user1`. Should the permission/role be set for the test user?
<img width="657" alt="Screen Shot 2020-09-15 at 1 52 24 AM" src="https://user-images.githubusercontent.com/5334504/93120556-20267380-f6f6-11ea-91c1-3b73167425fc.png">

Reply:
These lines should solve this problem. Maybe I am missing some more configuration?

https://github.com/mattermost/mattermost-webapp/blob/1b1c83883b6b77cb39e9501c1e8644f61e407e2c/e2e/cypress/integration/integrations/slash_commands_spec.js#L25-L30

Reply:
Still failing on me. I think it has something to do with license. Could you check if you're running with license? In CI, we normally run with license.

Reply:
I will do a admin login there to make sure it has permissions and remove the config update.